install/install.md

```diff
@@ -98,7 +98,7 @@ ERROR: No matching distribution found for cudf-cu12
 ```
 Check the suggestions below for possible resolutions:
 - The pip index has moved from the initial experimental release! Ensure the correct `--extra-index-url=https://pypi.nvidia.com`
-- Only Python versions 3.10 and 3.11 are supported
+- Only Python versions 3.10, 3.11, and 3.12 are supported
```

Comment:
Should this be a change we make post-release?

Reply:
Good point.

Even better, how about just directing people to scroll back up to the install selector?

I just proposed that in https://github.com/rapidsai/docs/pull/539/commits/5715c9efeed05b7165ee51f2949d18a6c2320810.

Making the install selector the single source of truth for the range of supported versions would reduce the effort for these upgrades and the risk of this list ever being out of date.